packages/aws-cdk/lib/cli/cli-config.ts

```diff
@@ -44,7 +44,7 @@ export async function makeConfig(): Promise<CliConfig> {
       'unstable': { type: 'array', desc: 'Opt in to unstable features. The flag indicates that the scope and API of a feature might still change. Otherwise the feature is generally production ready and fully supported. Can be specified multiple times.', default: [] },
     },
     commands: {
-      list: {
+      'list': {
```

Comment:
Mmm how come?

Reply:
because `'cli-telemetry'` needs it and apparently some pretty print decided that every other command needed to follow the same convention